### DISCUSSION TITLE
get  sensitive of a point in FFD box

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1937] [General]
hello， if I have some points in FFD box，how can i get their sensitive according to sensitive of wing sensitive？

**Comment by bigfooted** [comment]
After computing the primal with SU2_CFD and the adjoint with SU2_CFD_AD, you can get the sensitivities by running SU2_DOT_AD.
This tutorial shows the steps:
https://su2code.github.io/tutorials/Static_CHT/

**Comment by Zcaic** [comment]
OK, Thanks !
